Routers/post-router.js

```diff
@@ -0,0 +1,108 @@
+const express = require('express')
+
+const postDB = require('../data/helpers/postDb')
+
+const router = express.Router();
+
+
+
+router.get('/', (req, res) => {
+postDB
+    .get()
+    .then(post => {
+        res.status(200).json(post)
+    })
+    .catch(error => {
+        res.status(400).json({
+            error: "couldnt get posts"
+        })
+    })
+
+})
+
+
+
+router.get('/:id', (req, res) => {
+    const id = req.params.id;
+postDB
+    .getById(id)
+    .then(post => {
+        res.status(200).json(post)
```

Comment:
We should really be checking if we get something back here and send back a 404 response with a message saying that that post was not found by that id.